examples/src/main/java/ExampleListMessagesFiltered.java

```diff
@@ -0,0 +1,42 @@
+import com.messagebird.MessageBirdClient;
+import com.messagebird.MessageBirdService;
+import com.messagebird.MessageBirdServiceImpl;
+import com.messagebird.exceptions.GeneralException;
+import com.messagebird.exceptions.UnauthorizedException;
+import com.messagebird.objects.MessageList;
+
+
+public class ExampleListMessagesFiltered {
+    public static void main(String[] args) {
+
+        if (args.length == 0) {
+            System.out.println("Please specify your access key example : java -jar <this jar file> test_accesskey");
+            return;
+        }
+
+        // First create your service object
+        final MessageBirdService wsr = new MessageBirdServiceImpl(args[0]);
+
+        // Add the service to the client
+        final MessageBirdClient messageBirdClient = new MessageBirdClient(wsr);
+
+        try {
+            // Get list of messages with offset and limit
+            System.out.println("Retrieving message list");
+            
+            // Create filters
+            Map<String, Object> filters = new LinkedHashMap<>();
+            filters.put("status", "scheduled");
+
+            final MessageList messageList = messageBirdClient.listMessagesFiltered(3, null, filters);
```

Comment:
I run this example by using my access token. I have 1 delivered SMS, so for testing, I updated filter as delivered instead of scheduled...>> filters.put("status", "delivered");
The result is below, there is no item. If you check example response, there should be items https://developers.messagebird.com/api/sms-messaging#list-messages ... FYI: I will double-check this with the old endpoints, then we can revise response value again.

_> Retrieving message list
> ListBase{offset=3, limit=20, totalCount=1, links=Links{first='https://rest.messagebird.com/messages/?offset=0&status=delivered', previous='https://rest.messagebird.com/messages/?offset=0&status=delivered', next='null', last='https://rest.messagebird.com/messages/?offset=0&status=delivered'}, items=[]}_